2.13.15.md

```diff
@@ -0,0 +1,75 @@
+# Scala 2.13.15 (DRAFT)
+
+The Scala team at Lightbend is pleased to announce Scala 2.13.15.
+
+The following changes are highlights of this release:
+
+### Breaking changes
+
+These changes are especially likely to require user attention when upgrading:
```

Comment:
This is like "drinking coffee doubles your risk", where the risk is vanishingly small.

These changes to compiler behavior may require tweaks to build files or plugins:

Reply:
hmm.... I'm not sure this is an improvement, as "plugins" would have to be "compiler plugins" (to distinguish from sbt plugins), at which point it's reiterating the bullets themselves, which is why I went with "user attention"

I think I understand the motivation to tone down "especially likely" to "may". But actually all of the changes highlighted in the release notes may require user attention when upgrading.

tell you what, I'll just drop the entire sentence. I'll push that, but we can keep discussing

Reply:
It's like the "guess the disease" column in the newspaper. "Don't panic, but if you experience these strange symptoms, you may have this condition."

Reply:
after some further thought, I added a "Most plugins are not affected" bullet

Reply:
thank-you, doctor.

Reply:
I took a bullet for the team

Reply:
I'll take a Bulleit.